schemas/applications/schema/joint_positions.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Joint Positions",
+  "description": "A description of the joint configurations available in the application",
+  "type": "object",
+  "additionalProperties": false,
+  "patternProperties": {
+    "^[a-z]([a-z0-9_]?[a-z0-9])*$": {
+      "title": "Joint Positions",
+      "description": "The names and positions of a static joint configuration in the application",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "names": {
```

Comment:
`names` enough or do we need `joint_names`?

Reply:
I would prefer `joint_names`